approve.php

```diff
@@ -0,0 +1,91 @@
+<?php
+// Start session
+session_start();
+
+require_once('header.php');
+
+// Connect to the database
+$hostname = "localhost"; 
+$database = "fgp";
+$username = "fgp";
+$password = "fgp";
+
+$connection = mysqli_connect($hostname, $username, $password, $database);
+
+// Check if the connection was successful
+if (!$connection) {
+    die("Connection failed: " . mysqli_connect_error());
+}
+
```

Comment:
We might want to move this function to the dbpersons file to keep the code better organized